core/templates/dev/head/pages/exploration-player-page/layout-directives/learner-view-info.directive.ts

```diff
@@ -154,6 +154,16 @@ angular.module('oppia').directive('learnerViewInfo', [
                   $scope.infoCardBackgroundCss = {
                     'background-color': expInfo.thumbnail_bg_color
                   };
+                  $scope.titleWrapper = function() {
+                    var titleHeight =
+                    document.querySelectorAll(
+                      '.oppia-info-card-logo-thumbnail')[0].clientWidth - 20;
+                    var titleCss = {
+                      'word-wrap': 'break-word',
+                      width: titleHeight.toString()
```

Comment:
Doesn't `width` need to be in single quotes? If not, maybe remove  quotes from `word-wrap`?

Reply:
Yeah I added the `width` in quotes initially but it showed up as a lint error so I changed it

Reply:
@aks681 , I tried adding quotes to `width` it gives the following`error  Unnecessarily quoted property 'width' found  quote-props`. Also removing quotes from `word-wrap` causes error in angular which says `You may need an additional loader to handle the result of these loaders.
`. What should I do?

Reply:
Oh, then I guess this is fine. Thanks for checking!